### ISSUE TITLE
So I've had a bunch of issues throughout the process, but it still worked!

### ISSUE THREAD

**Issue by bdodroid** [filiphsandstrom/surface-duo#1]
So the big ones I was having a where issues trying to run ./globalify.sh
It would return permission denied even though I was rooted and su
I was able to run it via #sh globalify.sh 
but then it got a ton of "inaccessable or not found" errors through-out.

I ended up manually running each line of the globalify.sh script in adb shell and it worked just fine so I have no idea why it hated running the script directly.

After running all those commands manually, I reboot into recovery but kept getting "adb device unauthorized" and it wouldn't let me side load.

I eventually found a comment on a forum post from over a year ago that mentioned that when you see the little Android with the words "no command" 
you need to hold down the power button, hold down the vol up button, 
and let go of the power button to get the actual recovery options to show up.
Then you can go to "Update by adb" and then you can use the side load command.

Even then I got an error at 94% saying "adb: failed to read command: No error" but the output on the phone said 2/2 complete so I just rebooted and everything is working and I'm on android 11!

Hopefully this helps anyone else struggling. 

Anyway, thanks for your work on getting Android 11 on At&T model! Much appreciated!

Now its time to play with android 11 and see if it makes any difference on my phone lol

**Comment by rbeesley** [comment]
Can confirm. Even with su, you cannot chmod +x the globalify.sh. I also applied each a/b img, one by one.

Then I ran into the adb device unauthorized and while I was considering reimaging A10-ATT because I thought maybe I had broken things badly, I found out how to put the bootloader in "adb sideload" as part of the recovery process.

And then I hit this: `serving: '.\ota_b1-11-customer_gen_2021.1027.156.zip'  (~94%)    adb: failed to read command: No error`

The device says: `Install from ADB completed with status 0.`

I then performed a wipe data/factory reset and let it reboot. Success. I'm going through OOBE right now.
